index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="body-wrapper">
+        <nav class="header-flex">
+            <a class="logo" href=""><h5>Omar Mohammad</h5></a><span>
+            <img class="union" src="./assets/Union.png" alt='union-sign'/></span>
+        </nav>
```

Comment:
- [ ] I suggest using HTML5 semantic tags (e.g., header, nav) over divs to Follow [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), so the main container carrying the header section should be a `<header>` tag. kindly fix that. 👌

Reply:
Noted and resolved. Thanks a lot.